sensors/utils/utils.go

```diff
@@ -0,0 +1,53 @@
+// Package utils contains helper functions for the sensor implementations.
+package utils
+
+import (
+	"context"
+	"image"
+
+	"github.com/edaniels/gostream"
+	"github.com/pkg/errors"
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/rimage"
+	"go.viam.com/rdk/utils"
+
+	"go.viam.com/slam/config"
+)
+
+// GetName returns the name of the sensor based on its index in the sensor array.
+func GetName(svcConfig *config.AttrConfig, index int) (string, error) {
+	if index > len(svcConfig.Sensors) {
+		return "", errors.New("index out of bounds")
+	}
+	return svcConfig.Sensors[index], nil
+}
```

Comment:
@tessavitabile @EshaMaharishi I deleted `sensors.Sensor` and moved the `GetName` helper function into utils. Does this look like a sufficient solution?

Reply:
yes, looks good!